kiwi/builder/disk.py

```diff
@@ -793,14 +790,17 @@ def _create_dracut_config(self):
 
     def _create_system_dracut_config(self):
```

Comment:
From my perspective the entire method can be deleted.

The _create_system_dracut_config only existed to create a dracut conf that omits the kiwi modules. Now that the kiwi modules are installed as noops that function is no longer needed

Reply:
removed